codersdk/quota.go

```diff
@@ -8,8 +8,8 @@ import (
 )
 
 type WorkspaceQuota struct {
-	UserWorkspaceCount int `json:"user_workspace_count"`
-	UserWorkspaceLimit int `json:"user_workspace_limit"`
+	CreditsConsumed int `json:"credits_consumed"`
```

Comment:
We aren't really using the "credits" terminology anywhere else, right? Seems a little odd to introduce at this point.

Reply:
Yeah I agree, it could just be "QuotaConsumed"?